### DISCUSSION TITLE
New icon

### DISCUSSION THREAD

**Discussion by jpomykala** [Sequel-Ace/Sequel-Ace#861] [Ideas] [answered]
Hey, I really admire that somebody took the lead over Sequel Pro, and if we could we make SequelAce icon a little bit more modern it would be really great!

@Kaspik could you send here the *.svg file so I could play around with it? :) 

Thanks
Jakub

**Comment by Kaspik** [comment, marked as answer]
Hey, I believe the current icon is Big Sur ready and feels quite good, doesn't it?

Anyways we had a lot of proposals before implementing this one, check the threads: https://github.com/Sequel-Ace/Sequel-Ace/issues/31 / https://github.com/Sequel-Ace/Sequel-Ace/pull/411 / https://github.com/Sequel-Ace/Sequel-Ace/pull/176 and many more.

If you want, the icon is here on Github in Artwork folder, but I don't think we are planning on changing it for now. :)

**Reply by jpomykala** [reply]
Oh I see. So thanks for the response Jakub!

**Comment by xiangbohua** [comment]
My coworkers feel the Sequel Pro icon was batter too.
You will get used to it
